model/src/main/kotlin/CopyrightFinding.kt

```diff
@@ -32,10 +32,8 @@ data class CopyrightFinding(
      * The text location where the copyright statement was found.
      */
     val location: TextLocation
-) : Comparable<CopyrightFinding> {
+) {
     companion object {
-        private val COMPARATOR = compareBy<CopyrightFinding>({ it.statement }, { it.location })
+        val COMPARATOR = compareBy<CopyrightFinding>({ it.statement }, { it.location })
```

Comment:
I'm ok with the code, but I believe we need to tweak the rationale in the commit message a bit as there are IMO some unclarities:

> For example, the comparators of `CopyrightFinding` and `LicenseFinding` both use the comparator of `TextLocation`.

Strictly speaking, not the *comparator* of `TextLocation` is used, but they rely on `TextLocation` to *implement* `Comparable`. That's why this is not a good argument to keep the constant vs the implementation.

> So, inlining the comarator of `TextLocation` would lead to redundancy.

My proposal was to "inline" the `COMPARATOR` constant of `CopyrightFinding`, not the one of `TextLocation`.

Anyway, here's the body that I'd propose:

    Only sort on serialization for human readability and reproducibility.
    While that allows to remove the implementation of the comparable interface,
    still keep the comparator constant for use in the converter and any place
    where `CopyrightFinding`s shall be shown to users.

Reply:
PS: Do you plan to do the same thing for `LicenseFinding`, too?

Reply:
> PS: Do you plan to do the same thing for `LicenseFinding`, too?

Sure. Will be the next PR.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/7004.